Add Heroicons mini

bacali95 opened the pull request:
This PR adds the mini version of the 2nd version of Heroicons, according to https://github.com/react-icons/react-icons/pull/590#issuecomment-1245519004 they don't look the same as the solid ones.

@nolanleung can you please take a look? thanks :smile:

bacali95 commented:
@corysimmons ping

corysimmons commented:
pong 🏓

razzeee commented:
This would also be great, as it updates to the latest version of the icons, which added some icons I would want to use.

bacali95 commented:
@kamijin-fanta I have rebased my branch and resolved all conflicts, I hope you can get some time to look into it. thanks :smile:

bacali95 commented:
@kamijin-fanta I have rebased my branch again and updated the icons to the latest version, it seems that my GH action job needs approval to run, can you please approve that and also review this PR?

haikhalfakhreez commented:
Thanks for the PR! This would also solve #656

bacali95 commented:
Closing this because it was resolved by #745
